scripts/workers/code_upload_submission_worker.py

```diff
@@ -67,6 +111,77 @@ def get_submission_meta_update_curl(submission_pk):
     return curl_request
 
 
+def get_static_code_upload_no_submission_file_curl(
+    challenge_pk, phase_pk, submission_pk
+):
+    url = "{}/api/jobs/challenge/{}/update_submission/".format(
```

Comment:
@savish28 why are we making curl request from the worker? This is not the approach we discussed. The curl requests will be written in `make_submission.sh` we will only pass environment variables to the container. The curl request will use environment variable. Code upload worker shouldn't substitute curl params and share fixed curl with the submission job.